sdk/tables/azure-data-tables/tests/test_encoder.py

```diff
@@ -594,18 +594,9 @@ def test_encoder_create_entity_atypical_values(
             # Invalid int32 and int64 values
             max_int64 = 9223372036854775807
             test_entity = {"PartitionKey": "PK1", "RowKey": "RK1", "Data": int((max_int64 + 1) * 1000)}
-            expected_entity = test_entity
             with pytest.raises(TypeError) as error:
-                _check_backcompat(test_entity, expected_entity)
+                client.create_entity(test_entity)
             assert "is too large to be cast to" in str(error.value)
-            resp = client.create_entity(
```

Comment:
Why did we remove all the verification code from these tests?

Reply:
The `create_entity` call will fail in encoding  with this `test_entity`, we can't verify the request and response.